examples/api-samples/src/browser/api-samples-contribution.ts

```diff
@@ -16,8 +16,9 @@
 
 import { injectable, inject } from 'inversify';
 import { Command, CommandContribution, CommandRegistry, CommandHandler } from '@theia/core';
-import { FrontendApplicationContribution } from '@theia/core/lib/browser';
+import { AbstractViewContribution, FrontendApplicationContribution } from '@theia/core/lib/browser';
 import { SampleDynamicLabelProviderContribution } from './sample-dynamic-label-provider-contribution';
+import { SampleView } from '@theia/api-samples/lib/browser/sample-view';
```

Comment:
Here I get
```
error TS2307: Cannot find module '@theia/api-samples/lib/browser/sample-view'.
```
when I run `yarn`. Did I miss something?

Reply:
Same error in Travis CI:
https://travis-ci.com/eclipse-theia/theia/jobs/294291205#L729

Reply:
This import should be

`import { SampleView } from './sample-view';`

When importing from the same package one must use relative paths, picking up the ts(x) file from the src folder.  The lib folder and the js files will not exist until the package is compiled.

Reply:
'SampleView' should have a more specific name.  Many sample views may be added to this package over time.  Perhaps ExampleUnclosableView (as 'example' and not 'sample' is being used in this package, we should also be keeping that consistent).  See https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#unique-names

Reply:
I don't know what happened with the import, Webstorm imported it correctly in `api-samples-frontend-module.ts` but not in `api-samples-contribution.ts`... I should have doubled checked.